lib/config.js

```diff
@@ -36,6 +37,18 @@ class Config {
     this.tryOrWarn(() => {
       this.features = loadFeatures(this.paths().features);
     });
+    if (this.filters.length) {
+      this.filterFeatures();
+    }
+  }
+
+  filterFeatures() {
+    let filters = this.filters.map((path) => {
+      return this.base + '/' + path;
```

Comment:
should we run a `path.resolve` here? It would give us absolute paths which will play nicely when doing comparisons, like the `includes` below